liquid.py

```diff
@@ -242,16 +266,28 @@ def deploy():
     consul.set_kv('liquid_domain', liquid_domain)
     consul.set_kv('liquid_debug', liquid_debug)
 
-    for _, path in JOBS:
+    for job, path in JOBS:
+        print(f'Starting {job}...')
         runjob(path)
 
+    collections = get_config(None, 'collection:')
+    for name, settings in collections:
+        print(f'Starting collection-{name}...')
+        run_collection_job(name, settings)
```

Comment:
Same as below, let's have a single loop that calls `runjob`. This will be easier if `runjob` accepts the hcl job content instead of its path.